pkgs/applications/window-managers/phosh/fix-paths.patch

```diff
@@ -0,0 +1,10 @@
+--- a/data/phosh.in
++++ b/data/phosh.in
+@@ -1,6 +1,6 @@
+ #!/bin/sh
+
+-COMPOSITOR="/usr/bin/phoc"
++COMPOSITOR="@phoc@/bin/phoc"
```

Comment:
This should ideally be fixed upstream. Does the phoc package list the path in the `.pc` file?

Reply:
> Does the phoc package list the path in the .pc file?

I don't know what's the `.pc` file :(

Reply:
A `pkg-config` file; usually located in `$dev/lib/pkg-config` or `$dev/share/pkg-config`, it tells programs where to find dependencies. Looks like there is not one for phoc in the repository, though.

Reply:
AFAIK, pkg-config is used for library but `phoc` it isn't so I'm not sure what to do and how to do it.

Reply:
I don't think this is necessary, they properly use `@bindir@` later in the file:

https://source.puri.sm/Librem5/phosh/-/blob/8616d62239a20b3702bf86f38c59804f5ac1e44a/data/phosh.in#L48-50

Reply:
`phosh`’s `bindir` ≠ `phoc`’s bindir on Nix.